ls1intum/Artemis
# Fix Checklist transient problems

### Checklist

### Motivation and Context


Currently there is an issue that 2 transient properties are missing in the servers response for the exam checklist. In this PR we fix this. 

### Description

We add the necessary properties to the server call. 

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration -> Course -> Exam -> Checklist
3. Check that the number of participants for exercises is correct and that number of registered students is correct. 

### Test Coverage




### Screenshots
